pages/common/moro.md

```diff
@@ -1,23 +1,31 @@
-# Moro
+# moro
 
 > Track work time.
 
-- Run moro once right after starting the working day, and once more before leaving work:
+- Invoke `moro` without parameters, to set the current time as the start of the working day:
 
 `moro`
 
-- To adjust the start of the working day:
+- Specify a custom time for the start of the working day:
 
 `moro hi {{09:30}}`
 
-- To adjust the end of the working day:
+- Invoke `moro` without parameters a second time, to set the current time at the end of the working day:
+
+`moro`
+
+- Specify a custom time for the end of the working day:
 
 `moro bye {{17:30}}`
 
-- To add a note on the working day:
+- Add a note on the current working day:
 
 `moro note {{3 hours on project Foo}}`
 
-- To see the full report of all records and notes:
+- Invoke `moro` without parameters a third time, to show a report of the current working day:
```

Comment:
Can we update this to just `moro report` ? And say - `Show a report of the current working day`.

This is much clearer and avoids the confusion of trying to recall when does `moro` record the time, and when will it show a report.